pkg/helm/helm.go

```diff
@@ -78,13 +82,23 @@ func (h *Helm) FindVersions() error {
 	return err
 }
 
-// getReleasesVersionTwo retrieves helm 2 releases from ConfigMaps
+// getReleasesVersionTwo retrieves helm 2 releases from ConfigMaps or Secrets
 func (h *Helm) getReleasesVersionTwo() error {
+	var helmClient *helmstoragev2.Storage
 	if h.Version != "2" {
 		return fmt.Errorf("helm 2 function called without helm 2 version set")
 	}
-	hcm := driverv2.NewConfigMaps(h.Kube.Client.CoreV1().ConfigMaps(""))
-	helmClient := helmstoragev2.Init(hcm)
+	// helm store it's data in configmaps by default, but can be configured
+	// to use secrets instead (v2 only)
```

Comment:
No need for these comments, I think it's explained fine by the method comment on line 85